audit/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "lighthouse",
```

Comment:
and this should be `accessibilty-audit` too

Reply:
done